applications/nrf5340_audio/README.rst

```diff
@@ -1044,7 +1015,8 @@ After building the files for the development kit you want to program, complete t
 
       nrfjprog --program build/zephyr/zephyr.hex --coprocessor CP_APPLICATION --sectorerase -r
 
-   In this example, :file:`build/zephyr/zephyr.hex` is the HEX binary file for the application core.
+   :file:`build/zephyr/zephyr.hex` is the HEX binary file for the application core.
+   If a custom build folder is specified the path to this folder must be used in stead of :file:`build/`.
```

Comment:
```suggestion
   In this command, :file:`build/zephyr/zephyr.hex` is the HEX binary file for the application core.
   If a custom build folder is specified, the path to this folder must be used instead of :file:`build/`.
```

Reply:
Changed to suggested